src/Parser/TypeParser.php

```diff
@@ -83,6 +106,18 @@ public function enrichWithAttributes(TokenIterator $tokens, Ast\Node $type, int
 		return $type;
 	}
 
+	/**
+	 * @param list<Ast\Comment> $comments
+	 */
+	public function appendComments(Ast\Node $type, array $comments = []): Ast\Node
```

Comment:
1) should be private
2) I dislike optional parameters, make it required

Reply:
Absolutely. cee33d9